src/runtime_src/core/pcie/driver/linux/xocl/mgmtpf/mgmt-core.c

```diff
@@ -1272,6 +1272,16 @@ static int xclmgmt_probe(struct pci_dev *pdev, const struct pci_device_id *id)
 	(void) xocl_subdev_create_by_level(lro, XOCL_SUBDEV_LEVEL_BLD);
 	(void) xocl_subdev_create_vsec_devs(lro);
 
+	if (XOCL_DSA_IS_VERSAL(lro)) {
+		struct xocl_subdev_info subdev_info = XOCL_DEVINFO_XFER_MGMT_VERSAL;
+		xocl_info(&pdev->dev,
+			"Should be in VSEC, but now probe static Start 0x%llx",
```

Comment:
I would suggest removing "Should be in VSEC" here to avoid somebody who is greping dmesg nervous.